tests/Cms/Users/UserActionsTest.php

```diff
@@ -51,6 +51,14 @@ public function testChangeEmail()
         $this->assertEquals('another@domain.com', $user->email());
     }
 
+    public function testChangeEmailContainsUmlaut()
+    {
+        $user = $this->app->user('editor@domain.com');
+        $user = $user->changeEmail('test@exämple.com');
+
+        $this->assertEquals('test@exämple.com', $user->email());
+    }
+
```

Comment:
There should also be an assertion for the case where the input is a Punycode string:

```php
$user = $user->changeEmail('test@xn--tst-qla.com');
$this->assertSame('test@täst.com', $user->email());
```

Please also use `$this->assertSame()` for new tests. We are in the process of migrating `assertEquals()` to it.